flutter-studio/src/io/flutter/project/FlutterProjectCreator.java

```diff
@@ -135,7 +193,7 @@ public ModuleWizardStep modifySettingsStep(@NotNull SettingsStep settingsStep) {
       }
     };
     builder.setName(myModel.projectName().get());
-    builder.setModuleFilePath(FileUtil.toSystemIndependentName(contentRoot) + "/" + moduleName + ModuleFileType.DOT_DEFAULT_EXTENSION);
+    builder.setModuleFilePath(FileUtilRt.toSystemIndependentName(contentRoot) + "/" + moduleName + ModuleFileType.DOT_DEFAULT_EXTENSION);
```

Comment:
Maybe we can just replace this with `toSystemIndependentName` since it's statically imported above?

Reply:
Good catch. Next PR